numpy/_core/src/common/ufunc_override.c

```diff
@@ -99,12 +100,11 @@ PyUFuncOverride_GetOutObjects(PyObject *kwds, PyObject **out_kwd_obj, PyObject *
         *out_kwd_obj = NULL;
         return -1;
     }
-    /* borrowed reference */
-    *out_kwd_obj = _PyDict_GetItemStringWithError(kwds, "out");
-    if (*out_kwd_obj == NULL) {
-        if (PyErr_Occurred()) {
-            return -1;
-        }
+    int result = PyDict_GetItemStringRef(kwds, "out", out_kwd_obj);
```

Comment:
I'm not sure of the refcount if result is 1.

Reply:
Thanks for spotting this, totally ignored that case.